src/main/steps/upload-documents/content.test.ts

```diff
@@ -1,47 +1,49 @@
-import { cy, en, form, generateContent } from './content';
+import { cy, en, form, generateContent, getErrors } from './content';
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 const englishContent = () => ({
-  serviceName: 'DSS Update Case',
-  title: 'Upload documents',
+  serviceName: 'Update an existing First-tier Tribunal case',
+  title: 'Add information to a case',
   paragraph1:
-    'You can upload additional documents, files or other evidence to support your case. You can also add information below that is relevant to the case.',
-  uploadedDocumentsSubTitle: 'Uploaded documents can include:',
+    'Please provide any additional relevant information that you would like to add to this case. Don\'t include any personal financial information, such as your banking details.',
+  additionalDocumentsHeading: 'Additional documents',
+  paragraph2: 'You may want to submit other documents, files, or evidence that provides the Tribunal with more information about your case.',
+  paragraph3: 'This will depend on the nature of your appeal, application or claim. Specific examples can be found in your completed tribunal form.',
+  uploadedDocumentsSubTitle: 'These may include:',
   uploadDocumentsBullets: [
     'copies of any relevant correspondence or documentation',
+    'audio or video recordings',
     'written statements',
     'related reports',
   ],
-  caseRelevancySubTitle: 'For each document uploaded you must state why it is relevant to the case.',
+  caseRelevancySubTitle: 'For each piece of additional information or evidence you are submitting, please state why it is relevant to your case.',
   fileUploadRequirementsLabel: 'File upload requirements',
   uploadHelpBullets: [
-    'File formats: MS Word, MS Excel, PDF, JPG, PNG, TXT, RTF',
+    'File formats: MS Word, MS Excel, PDF, JPG, PNG, TXT, RTF, MP4, MP3',
     'File size per document: up to 20 megabytes (MB)',
+    'File size per multimedia file: up to 30 megabytes (MB)',
     'Files cannot be password protected',
   ],
   guideText:
-    'You cannot upload executable (.exe) or zip files because of virus risks. If you delete an uploaded file, you also delete the file description.',
+    'You can\'t upload executable (.exe), zip or other archive files due to virus risks.',
```

Comment:
We're trying to remain consistent with dss create case and the wireframes provided in the ticket. This an example in dss create case.

<img width="595" alt="Screenshot 2024-02-20 at 10 50 47" src="https://github.com/hmcts/sptribs-dss-update-case-web/assets/51362727/6be30347-4e11-407e-8f49-cdc63b0119d4">

Think this could be something that we improve on both dss create case and dss update case in the future.